osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/dlmsobjectconfig/AttributeAddressForProfile.java

```diff
@@ -44,6 +44,21 @@ public Integer getIndex(final DlmsObjectType type, final Integer attributeId) {
     return null;
   }
 
+  public Integer getIndex(final DlmsObjectType type, final Integer attributeId, final int channel) {
+    int index = 0;
+
+    for (final DlmsCaptureObject object : this.selectedObjects) {
+      if (object.getRelatedObject().getType().equals(type)
```

Comment:
is dit NPE safe? object.getRelatedObject()?

Reply:
A DlmsCaptureObject should always have a related object

Reply:
Ok